mods/ts/rules/gdi-support.yaml

```diff
@@ -137,10 +137,19 @@ GACTWR:
 			tower.rocket: !build-incomplete && !tower.rocket && !tower.vulcan && !tower.sam
 			tower.sam: !build-incomplete && !tower.vulcan && !tower.rocket && !tower.sam
 	ProvidesPrerequisite@buildingname:
+	ProvidesPrerequisite@pluggable:
+		RequiresCondition: !(tower.vulcan || tower.rocket || tower.sam)
```

Comment:
I'm on the fence on this one since it would be more compact than distributing the negation if there were more than three condition variables but the negation is distributed above:
```yaml
			tower.vulcan: !build-incomplete && !tower.vulcan && !tower.rocket && !tower.sam
			tower.rocket: !build-incomplete && !tower.rocket && !tower.vulcan && !tower.sam
			tower.sam: !build-incomplete && !tower.vulcan && !tower.rocket && !tower.sam
```
Factoring out the negations above is also an option.

Reply:
@abcdefg30 mentioned preferring the distributed form from above on #openra IRC channel.